package chap06;

public class ExceptionTest {

	public static void main(String[] args) {
		int number = 100;
		int result = 0;
		
		try {
			
		} catch (Exception e) {
			System.out.println(number/0);
		}
		
	}

}
